cmds/modules/netlightd/main.go

```diff
@@ -107,18 +108,29 @@ func action(cli *cli.Context) error {
 		return fmt.Errorf("failed to apply host nft rules: %w", err)
 	}
 	rules.Close()
-	bridge, err := netlight.CreateNDMZBridge()
+	_, err = netlight.CreateNDMZBridge()
 	if err != nil {
 		return fmt.Errorf("failed to create ndmz bridge: %w", err)
 	}
 
-	_, err = resource.Create("dmz", bridge, &net.IPNet{
-		IP:   net.ParseIP("100.127.0.2"),
-		Mask: net.CIDRMask(16, 32),
-	}, netlight.NDMZGwIP, nil, myceliumSeedFromIdentity(identity.PrivateKey(cli.Context)))
-
+	// create mycelium for host
+	hostMyCelium := "hmycelium"
+	myBr := "mdmz"
```

Comment:
what is the m?

Reply:
it is for mycelium, I kept the old naming but we can rename it to `hmy-br` (host mycelium bridge) for example  will be better